### PR TITLE
SFCC-41:  Fix bug - image not being returned to UI

### PR DESCRIPTION
**Pull request by kenredvan**

### CODE REVIEW

File: cartridges/int_stickyio_sfra/cartridge/scripts/subscription/subscriptionHelpers.js [hunk 1]
```diff
@@ -140,7 +140,7 @@ function getSubscriptions(currentCustomer, querystring, locale, billingModels) {
             orders.push(orderModel);
         }
     }
-
+
```
Comment (asecaida) [inline cartridges/int_stickyio_sfra/cartridge/scripts/subscription/subscriptionHelpers.js:143]: remove these extra spaces please

File: cartridges/int_stickyio_sfra/cartridge/scripts/subscription/subscriptionHelpers.js [hunk 2]
```diff
@@ -153,6 +153,7 @@ function getSubscriptions(currentCustomer, querystring, locale, billingModels) {
         var thisOrder = OrderMgr.getOrder(orders[i].orderNumber, orders[i].orderToken);
         customerID = thisOrder.getCustomer().ID; // this gets overwritten every time, but that's ok
         var orderShipments = thisOrder.getShipments();
+        orderModel = orders[i];
```
Comment (asecaida) [inline cartridges/int_stickyio_sfra/cartridge/scripts/subscription/subscriptionHelpers.js:156]: remove the extra spaces at the end of the line please

**Review by asecaida — CHANGES_REQUESTED** [review]

File: cartridges/int_stickyio_sfra/cartridge/scripts/subscription/subscriptionHelpers.js (hunk 2 above)
Comment (kenredvan) [reply cartridges/int_stickyio_sfra/cartridge/scripts/subscription/subscriptionHelpers.js]: done

File: cartridges/int_stickyio_sfra/cartridge/scripts/subscription/subscriptionHelpers.js (hunk 1 above)
Comment (kenredvan) [reply cartridges/int_stickyio_sfra/cartridge/scripts/subscription/subscriptionHelpers.js]: done

**Review by asecaida — APPROVED** [review]

**Review by YuloVS — APPROVED** [review]

**Pull request merged**